forum/investors/models.py

```diff
@@ -1,11 +1,15 @@
 from django.db import models
+from django.contrib.auth import get_user_model
+from startups.models import Startup
 from startups.models import Location
 import uuid
 from django.core.validators import MinValueValidator
 from django.core.exceptions import ValidationError
 
 from users.models import User
 
+User = get_user_model()
```

Comment:
You’re importing `User = get_user_model()`, but also importing the `User model` from `users.models`. Ensure consistency by using either `get_user_model()` or a direct import from `users.models` but avoid both.